src/widgets/gauge.rs

```diff
@@ -12,9 +12,9 @@ use crate::{
 ///
 /// A `Gauge` renders a bar filled according to the value given to [`Gauge::percent`] or
 /// [`Gauge::ratio`]. The bar width and height are defined by the [`Rect`] it is
-/// [rendered](Widget::render) in.  
```

Comment:
My editor autoremoved trailing whitespaces. I'm not sure if they are needed 🤔

Reply:
They are needed for the newlines, you should probably add them back :D

With trailing whitespaces:

![image](https://github.com/ratatui-org/ratatui/assets/24392180/24abcf54-6e4a-4e56-b388-1f38d612a841)

Without trailing whitespaces:

![image](https://github.com/ratatui-org/ratatui/assets/24392180/8b3c4293-12b8-49de-9fbf-9c4408913bc6)

Reply:
As a general rule, I strongly prefer to avoid end of line spaces like this as editors and other tooling often bork on them, and they're invisible to users until the output. Use a blank line instead here and a `<br>` in markdown if you really need a newline rather than a new paragraph.

Reply:
I've added additional line and I think it fixed the issue. Now it looks like this:
<img width="989" alt="image" src="https://github.com/ratatui-org/ratatui/assets/5149215/7bf2575b-f11e-4d7e-b0bd-5766645e698d">